src/Validator/Constraints/UniqueTitleValidator.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Validator\Constraints;
+
+use App\Repository\TalkRepository;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class UniqueTitleValidator extends ConstraintValidator
+{
+    public function __construct(
+        private TalkRepository $talkRepository,
+    ) {
+    }
+
+    /**
+     * @param string      $formTitle
+     * @param UniqueTitle $constraint
+     */
+    public function validate($formTitle, Constraint $constraint): void
+    {
+        if (!$formTitle || '' === $formTitle) {
```

Comment:
```suggestion
        if (!$formTitle) {
```

Reply:
These changes were made in a previous PR and this one was not yet rebased, this is fixed in #210